gobblin-core/src/main/java/gobblin/source/extractor/extract/jdbc/JdbcExtractor.java

```diff
@@ -964,20 +953,31 @@ public long getCount(CommandOutput<?, ?> response) throws RecordCountException {
       throw new DataRecordException("Failed to get records from MySql; error - " + e.getMessage(), e);
     }
   }
-  
-  private Set<String> getBlobTypeColumnNames(ResultSetMetaData resultsetMetadata) throws SQLException{
-    Set<String> result = new HashSet<String>();
-    int columnCount = resultsetMetadata.getColumnCount();
-    for (int i = 1; i <= columnCount; i++) {
-      // for Mysql Longblob and blob type, java type is Types.LONGVARBINARY
-      // for Mysql binary type, java type is Types.Binary
-      if (resultsetMetadata.getColumnType(i) == Types.LONGVARBINARY ||
-          resultsetMetadata.getColumnType(i) == Types.BINARY){
-        // the actual value will be set later
-        result.add(resultsetMetadata.getColumnName(i));
-      }
+
+  /*
+   * For Blob data, need to get the bytes and use base64 encoding to encode the byte[]
+   * When reading from the String, need to use base64 decoder
+   *     String tmp = ... ( get the String value )
+   *     byte[] foo = Base64.decodeBase64(tmp);
+   */
+  private String readBlobAsString(Blob logBlob) throws SQLException {
+    byte[] ba= logBlob.getBytes(1L, (int)(logBlob.length()));
```

Comment:
Is it possible for `getBytes` to return a `null`? BTW: need one space before `=`.